README.md

```diff
@@ -19,8 +19,13 @@ The main objective is to showcase that a user can take a trained model, use a pi
 1. Configure the default Identity Provider
 1. Install Red Hat Advanced Cluster Management
 1. Register the clusters
-   * Core - Cluster host the ODH Core components that will be used in the MLOps Engineer workflow to train, build and push the model
-   * Near Edge - Cluster that will host the running model at the edge.  This is the target environment after a new model is available for use
+   [ACM Application](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/applications/managing-applications) manifests are located in [acm/registration](acm/registration) to register and configure the target environments required for the AI at the Edge use cases.  The files ca be applied to the ACM hub cluser manually:
+   ```
+   $ kustomize build acm/registration | oc apply -f -
```

Comment:
Is the `kustomize` part necessary, when there's no `kustomization.yaml` included here? Actually, it doesn't work without it:

```
$ kustomize build acm/registration | oc apply -f -
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/var/home/gryan/src/github.com/opendatahub-io/ai-edge/acm/registration'
error: no objects passed to apply
```

* Without `kustomization.yaml`, this will work:
```suggestion
   $ oc apply -f acm/registration/
```

* Or if you add a `kustomization.yaml`, this might be simpler, now that `kustomize` is built into `kubectl`/`oc`:
```suggestion
   $ oc apply -k acm/registration/
```

Reply:
I forgot to mark this as a draft since the kustomization.yaml was missing from the initial PR.

`oc apply -k <kustomize manifest>` is fine.  I have muscle  memory using `kustomize build` for testing compatibility with different kustomize versions for the new ODH manifests